src/test/java/pl/apanowicz/demoapp/api/ProductEndpointTests.java

```diff
@@ -0,0 +1,149 @@
+package pl.apanowicz.demoapp.api;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import pl.apanowicz.demoapp.DemoappApplicationTests;
+import pl.apanowicz.demoapp.domain.ProductFacade;
+import pl.apanowicz.demoapp.domain.ProductRequestDto;
+import pl.apanowicz.demoapp.domain.ProductResponseDto;
+
+
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class ProductEndpointTests extends DemoappApplicationTests {
+
+    @Autowired
+    ProductFacade productFacade;
+
+    @Test
+    public void shouldGetExistingProduct() {
+        ProductRequestDto requestDto = new ProductRequestDto("product");
+        ProductResponseDto existingProduct = productFacade.create(requestDto);
+        final String url = "http://localhost:" + this.port + "/api/v1/products/" + existingProduct.getId();
```

Comment:
Wydaje się, że potrzebujemy podobnego typu informacji (url) na poziomie co najmniej kilku testów. Może warto to wydzielić do zmiennej dostępnej w ramach tej klasy?